### ISSUE TITLE
Value wrapping for drag widget

### ISSUE THREAD

**Issue by Rageking8** [ocornut/imgui#7749]
### Version/Branch of Dear ImGui:

Version 1.90.6, Branch: docking

### Back-ends:

imgui_impl_glfw.cpp + imgui_impl_opengl3.cpp

### Compiler, OS:

Windows 11 + VS2022

### Details:

**My Question:**

How to make a drag widget that has value wrapping? For example if the range of the drag widget is [0.f, 1.f] and the user drags pass either end of the range, the value would wrap to the other end.

**Comment by ocornut** [comment]
It’s not currently trivially possible but we could add the feature. We may need some form of “resistance” code to make it easier to reach the limit, where an additional movement is required to wrap, but make that feature also optional.

**Comment by ocornut** [comment]
Added `ImGuisliderFlags_WrapAround` in 0403096

```cpp
ImGui::DragInt("drag int wrap 100..200", &i3, 1, 100, 200, "%d", ImGuisliderFlags_WrapAround);
```
```cpp
ImGui::DragFloat("drag wrap -1..1", &f3, 0.005f, -1.0f, 1.0f, NULL, ImGuisliderFlags_WrapAround);
```

It's a little ambiguous whether both limits should be considered as the same value for integer values. Since clamp limits are inclusive, it seems to make sense when using integers that limits are not same value, so with 100..200 the next value after 200 is 100 and not 101. There isn't a natural step for floating points so I assuming that are the same.

Let me know if that works for you!

**Comment by Rageking8** [comment]
@ocornut Thank you so much for such a swift implementation, highly appreciate it. However, while looking through the commit diff, I realized that for the new flag `ImGuisliderFlags_WrapAround` introduced, the casing of `slider` should have been `Slider` (capital "S").

If this is in fact unintended, do let me know as I would love to create a PR for the fix. Thanks in advance.

**Comment by kevincoghlan** [comment]
@Rageking8 Yes, I suspect it was a typo. I put up a quick PR when I saw it (#7752).

**Comment by ocornut** [comment]
Fixed the typo now, thank you!
